docs/en/04_Changelogs/4.8.0.md

```diff
@@ -10,47 +10,58 @@
+- look at the [`silverstripe/graphql` pull requests](https://github.com/silverstripe/silverstripe-graphql/pulls) to see exactly what has changed.
+
+The resulting`silverstripe/graphql:^4` release is currently in an alpha stage.
 
 We are planning to require the use of `silverstripe/graphql:^4` in a future CMS 4.x release.
 
+#### Start using Silverstripe GraphQL v4 today
+
+We encourage everyone to get started working with GraphQL v4 today:
+- You can start by reading our new [GraphQL documentation](https://docs.silverstripe.org/en/4/developer_guides/graphql/).
+- You can provide [feedback](https://github.com/silverstripe/silverstripe-graphql/issues) and help us stabilise the module.
+
+If your project uses `silverstripe/recipe-cms`, it is still locked to `silverstripe/graphql:^3.5`.
+To use `silverstripe/graphql:^4`, you'll need to "inline" the `silverstripe/recipe-cms` requirements
+in your root `composer.json` and change `silverstripe/graphql` to `^4`.
+
+You can inline `silverstripe/recipe-cms` by running this command:
+```bash
+composer update-recipe silverstripe/recipe-cms
+```
+
+Alternatively, you can remove `silverstripe/recipe-cms` from your root `composer.json` and replace
+it with the the contents of the `composer.json` in `silverstripe/recipe-cms`.
+
+
 #### GraphQL 3 code moved to _legacy folder
 
 You will notice that many of the core modules now have a `_legacy` directory in their root.
+In order to facilitate the new module release, most GraphQL helper classes to drive the CMS UI
+have been marked as deprecated, and moved to `GraphQL/_legacy` folders within core modules.
 This is a temporary fix that we have applied to satisfy the competing interests of semver and
 PSR-4 autoloading.
 
-Because the semver contract precludes changing the fully-qualified names of any classes,
-we need to keep GraphQL 3 code untouched. But GraphQL 4 code often contains classes of similar
-or identical names, e.g. "ReadFilesResolver", or something to that effect. Further, this 
-backward compatibility creates a common demand for the canonical namespace "GraphQL.", e.g.
-`SilverStripe\AssetAdmin\GraphQL`. It didn't seem appropriate to put _old_ code in the "GraphQL"
-space, and _new_ code in a less obvious place, like `SilverStripe\AssetAdmin\GraphQL4`.
-
-A simple fix would have been to stuff all the GraphQL 3 code into `code/GraphQL/_legacy` and use `classmap` to shim the PSR-4 autoloader, but this is [no longer supported](https://blog.packagist.com/composer-2-0-is-now-available/#3-backwards-compatibility-breaks) in Composer 2.0.
+The GraphQL schema under `admin/graphql` has been altered slightly to enable forwards compatibility,
+with lower-camelcase field names, and a different filter argument structure.
+In the unlikely case that you are relying on those classes or queries directly,
+consider upgrading to `silverstripe/graphql:^4`.
 
```

Comment:
I removed this bit because it's getting a bit too far in the weeds and I don't think we should justify the detail of our implementation decision in our changelog. If people, think this is essential information that needs to be communicated to developers upgrading projects, than I'm fine with brining it back.

If there's a summary of why we did this somewhere, we could just link to it.